src/parser.cpp

```diff
@@ -318,23 +460,26 @@ void Parser::ParserImpl::loadModel(const ModelPtr &model, const std::string &inp
     std::vector<XmlNodePtr> connectionNodes;
     std::vector<XmlNodePtr> encapsulationNodes;
     while (childNode) {
-        if (childNode->isCellmlElement("component")) {
+        if (childNode->isCellml20Element("component") || (mTransformFrom1X && childNode->isCellml1XElement("component"))) {
```

Comment:
Because I could have a CellML 1.0 component inside a CellML 2.0 model.  Contrived, sure but possible.

Reply:
Contrived indeed, but then following the same contrived logic we should then have:
```c++
if ((!mTransformFrom1X && childNode->isCellml20Element("component"))
    || (mTransformFrom1X && childNode->isCellml1XElement("component"))) {
...
}
```
in case a CellML 1.X model has a CellML 2.0 component... :)

Reply:
[dcb9335](https://github.com/cellml/libcellml/pull/982/commits/dcb933538c32ea696fe8f1bf1a002b8c4d556ebc)